### DISCUSSION TITLE
Proxmox Cluster Creation fails on first run

### DISCUSSION THREAD

**Discussion by byly-ed** [lae/ansible-role-proxmox#199] [General]
Hello, everyone,
my company and I are trying to automate our Proxmox infrastructure and that's how we came across this project. First of all, a big compliment to all developers who have contributed to this project. In principle, everything works when I run the site.yml, but it gets stuck at the step: "Add node to Proxmox cluster". If I then stop the script and run it again, the script runs smoothly. I see that the second time I run the script, the "Copy PVE SSL certificate chain and key to /etc/ssl" step also copies the SSL certificates and everything seems to work after that. And if I add a new host to this cluster, the same thing happens again as at the beginning. The script runs on freshly installed Debian 11 systems. Is this error already known or can someone help me?

**Comment by lae** [comment]
Please post your role variables and the site.yml.

You may want to check the systemd journal (`sudo journalctl -xb`) for any errors as well when a node is being added.

**Comment by byly-ed** [comment]
Sure, i attach the files. 
[debug-output-server1.txt](https://github.com/lae/ansible-role-proxmox/files/9308487/debug-output-server1.txt)
[debug-output-server2.txt](https://github.com/lae/ansible-role-proxmox/files/9308488/debug-output-server2.txt)
[site.txt](https://github.com/lae/ansible-role-proxmox/files/9308489/site.txt)
[tc1.txt](https://github.com/lae/ansible-role-proxmox/files/9308506/tc1.txt)

**Reply by lae** [reply]
I'm not seeing anything overtly broken or incorrect here. Unfortunately, I can't help you further troubleshoot currently, but you may want to insert a [pause](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pause_module.html) before the add node task in your local cache of the role (likely `~/.ansible/roles/lae.proxmox/tasks/pve_add_node.yml`) and manually run the clustering command and see what happens.

I just successfully deployed a test cluster over three Debian 11 VMs without issue, so there shouldn't be an issue unless there's something else about your configuration or environment that could be stalling that step.
